mungegithub/submit-queue/www/script.js

```diff
@@ -235,15 +238,25 @@ function SQCntl(dataService, $interval, $location) {
           failedBuild = true;
       }
       obj.stability = '';
-      result.push(obj);
+      if (!obj.msg) {
+        result.push(obj);
+      } else {
+        if (obj.msg.includes('[nonblocking]')) {
+          obj.msg = ''
```

Comment:
I don't think there's a case where there's more to the message now, but that's good future proofing.  Fixed.